docs/en/operation/Basic-Logging.md

```diff
@@ -6,17 +6,34 @@ title: Basic Logging
 
 ## Introduction
 
-This page describes the basic logging function provided by Alluxio server processes (e.g., masters, workers and etc)
+This page describes the basic logging function provided by Alluxio server processes (e.g., masters, workers, and etc.)
 and application processes utilizing Alluxio clients (e.g., Spark or MapReduce jobs running on Alluxio).
 
-Alluxio logging is implemented using [`log4j`](https://logging.apache.org/log4j/) and thus
+Alluxio logging is implemented using [`log4j`](https://logging.apache.org/log4j/){:target="_blank"} and thus
 most of the configuration is done through modifying `log4j.properties`.
 
+The master and worker logs are useful to understand what the Alluxio Master and
+Workers are doing, especially when running into any issues. If you do not understand the error messages,
+search for them in the [Github issues](https://github.com/Alluxio/alluxio/issues){:target="_blank"},
+in the case the problem has been discussed before. 
+You can also join our [Slack channel](https://slackin.alluxio.io/) and seek help there.
+You can find more details about the Alluxio server logs [here]({{ '/en/operation/Basic-Logging.html#server-logs' | relativize_url }}).
+
+The client-side logs are helpful when Alluxio service is running but the client cannot connect to the servers.
+Alluxio client emits logging messages through log4j, so the location of the logs is determined by the client side
+log4j configuration used by the application.
+You can find more details about the client-side logs [here]({{ '/en/operation/Basic-Logging.html#application-logs' | relativize_url }}).
+
+The user logs in `${ALLUXIO_HOME}/logs/user/` are the logs from running Alluxio shell.
+Each user will have separate log files.
+
 ## Log Location
 
 ### Server Logs
 
 Log files for each individual Alluxio server process (e.g., master, worker, FUSE, proxy) can be found under `${ALLUXIO_HOME}/logs/`.
+They are named as `master.log`, `master.out`, `worker.log`, `worker.out`, `job_master.log`, `job_master.out`, 
```

Comment:
```suggestion
Each process should have two corresponding files ending with `.log` or `.out`, such as `worker.log` and `worker.out` for the worker process.
```
i wouldn't attempt to list out all the possible process .log and .out files since they could easily be changed.